src/java/boa/functions/BoaAstIntrinsics.java

```diff
@@ -209,9 +210,12 @@ private static void openMap() {
 		final Configuration conf = new Configuration();
 		try {
 			final FileSystem fs = FileSystem.get(conf);
-			final Path p = new Path("hdfs://boa-njt/",
```

Comment:
When that code is ready, as part of that pull request this code can be generalized. At this moment, let us proceed with this change.